ui/src/app/shared/service/service.ts

```diff
@@ -126,7 +126,7 @@ export class Service extends AbstractService {
     // this.notify(notification);
   }
 
-  public setCurrentComponent(currentPageTitle: string | { languageKey: string, interpolateParams?: Object }, activatedRoute: ActivatedRoute): Promise<Edge> {
+  public setCurrentComponent(currentPageTitle: string | { languageKey: string, interpolateParams?: {} }, activatedRoute: ActivatedRoute): Promise<Edge> {
```

Comment:
```{}``` is better than ```Object``` but still not good, anyway this method is deprecated and shouldnt be used in the future anyway